source/minimessage/format.rst

```diff
@@ -0,0 +1,272 @@
+.. _minimessage-format:
+
+Format
+======
+
+The Minimessage language uses tags. Everything you do will be defined with tags. Tags have a start tag and an end tag (the :mm:`<reset>` tag is an exception here).
+Start tags are mandatory (obviously), end tags aren't outside of ``strict`` mode.
+:mm:`<yellow>Hello <blue>World<yellow>!` and :mm:`<yellow>Hello <blue>World</blue>!` and even :mm:`<yellow>Hello </yellow><blue>World</blue><yellow>!</yellow>` all do the same.
+
+All tag names are case-insensitive to reduce the possibility for conflict, but we recommend keeping all tag names lowercase (or at the very least, being consistent).
```

Comment:
Is this fully true? I thought tags with non-lowercase letters were invalid, or is that only the case when defining them, not using them?

Reply:
That's only the case when defining them, so that tags can be case-insensitive at use time.